starboard/build/toolchain/cobalt_toolchains.gni

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build/config/win/visual_studio_version.gni")
+import("//build/toolchain/gcc_toolchain.gni")
+
+template("cobalt_clang_toolchain") {
+  forward_variables_from(invoker.variables,
```

Comment:
Let's forward this in the toolchain is possible (i.e. around line 28) as that's what'll actually use it, right now it's just implicitly captured

Reply:
done